lib/bump/cli/commands/deploy.rb

```diff
@@ -9,12 +9,13 @@ class Deploy < Base
         option :id, default: ENV.fetch("BUMP_ID", ""), desc: "Documentation public id"
         option :token, default: ENV.fetch("BUMP_TOKEN", ""), desc: "Documentation private token"
         option :specification, desc: "Specification of the definition"
+        option :validation, desc: "Validation mode: basic, strict"
 
-        def call(file:, id:, token:, specification: nil)
+        def call(file:, id:, token:, specification: nil, validation: nil)
```

Comment:
doing a `basic` validation by default is different from passing a `nil`?
If so, we should be more explicit about the available options by documenting/allowing the `none` / nil option alongside `basic` or `strict`
If no difference, why not going with `basic` as the default?

Reply:
Well, I wanted to have the same mechanism that for specification (= let the server decide), but maybe in this case it's better to have a default value directly in the client.